be/src/exprs/bitmap_functions.cpp

```diff
@@ -675,4 +675,60 @@ StatusOr<ColumnPtr> BitmapFunctions::bitmap_subset_in_range(FunctionContext* con
     return builder.build(ColumnHelper::is_all_const(columns));
 }
 
+StatusOr<ColumnPtr> BitmapFunctions::bitmap_to_binary(FunctionContext* context, const starrocks::Columns& columns) {
+    ColumnViewer<TYPE_OBJECT> viewer(columns[0]);
+
+    size_t size = columns[0]->size();
+    ColumnBuilder<TYPE_VARCHAR> builder(size);
+
+    std::unique_ptr<char[]> buf;
+    size_t cur_buf_size = 0;
```

Comment:
Why not use RawString to make things easier?
RawString will compute capacity automatically according to the input room requirements.

Reply:
Good suggestion, fixed.